### DISCUSSION TITLE
Clean install or...

### DISCUSSION THREAD

**Discussion by houstonbofh** [immense/Remotely#308] [Q&A]
I installed Remotely to evaluate it a little while ago, but before I could really do anything, you released the new version.  So now the question is should I upgrade my mostly empty install, or remove it and install fresh?  And if so, what is a clean removal?

**Comment by bitbound** [comment]
It's designed for continuous integration.  So you can just redeploy the new version over the old one to update.  As long as you've created a copy of `appsettings.json` and called it `appsettings.Production.json`, none of your settings will get overwritten.

The unattended agents will auto-update themselves the next time they connect to the server.  The installable quick support client will too.  The single-file EXE doesn't, though.

My test environment, where all mine and my family's computers are, gets redeployed on every commit.  When I'm really active with development, I get really close to my data cap from all of our machines constantly auto-updating their agents.  :D   The public server gets deployed over pretty frequently too.

**Reply by houstonbofh** [reply]
So just download this and run it on my current installation, not a clean install?  https://github.com/lucent-sea/Remotely/releases/download/v2021.04.18.2239/Remotely_Server_Installer  Or is the next version coming soon?

**Reply by bitbound** [reply]
Yup.  The first question it asks is whether to install the pre-built package.  If true, that will install the last full release, which is the one with the "old" UI.

If false, it will build from my master branch.  You can choose to pull the very latest changes by entering "master" for the reference, or you can enter a release tag to build from that.  I'm thinking of changing this to be an "pick this or that" selection between "preview" (i.e. latest changes on master) or "latest" (i.e. last full release).

The master branch is changing frequently right now since I'm tweaking things and fixing bugs in preparation for the full release.

But after the next full release, I'm going to stop working out of master, so it will be more stable.

Hopefully that makes sense.  :)   Let me know if you have any questions or feedback on the process.

**Reply by bitbound** [reply]
Also, the pre-built packages won't have your server URL embedded in the clients.

**Reply by sgluhov** [reply]
Hello I am successfully trying to use a custom-build package install, but I have an error durring installer:
I think Licence directory is not created properly during install and that breaks the script:
![remotely](https://user-images.githubusercontent.com/42271441/182001913-dd84a3b8-a18c-44f8-a415-0295a1a25109.png)

Should be quite easy to be fixed.
Please help!
